src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +534,55 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            if (e.BuildEventContext == null || e.Items == null)
+            {
+                return;
+            }
+            // node and project context ids for the propertyOutputMap key
+            int nodeID = -1;
```

Comment:
These are redundant now, so can you collapse definition/assignment?

But also there's a named constant for this:

```suggestion
            int nodeID = BuildEventContext.InvalidNodeId;
```